### PR TITLE
Automated Test - COMMERCE-6318 | master

### PR DESCRIPTION
**Pull request by lucasperj**

### CODE REVIEW

**Comment by lucasperj** [comment]
https://issues.liferay.com/browse/COMMERCE-6318 ; cc: @lorenzolapescara

**Comment by kristinonias** [comment]
ci:test:sf

**Comment by lorenzolapescara** [comment]
Review started

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase [hunk 1]
```diff
@@ -154,12 +154,62 @@ definition {
...
+
+		CommerceNavigator.gotoPortlet(
+			category = "Product Management",
+			portlet = "Products");
+
+		AppBuilderAdmin.searchByItem(itemName = "U-Joint");
+
+		Click(
+			key_entryName = "U-Joint",
+			locator1 = "CommerceEntry#TABLE_LIST_TITLE");
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase:171]:
```suggestion
		CommerceNavigator.gotoEntry(entryName = "U-Joint");
```

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by pedro-michel** [comment]
ci:test:sf

**Review by lorenzolapescara — APPROVED** [review]

**Comment by lorenzolapescara** [comment]
ci:test:commerce-functional

**Comment by lorenzolapescara** [comment]
ci:test:commerce-functional

**Comment by lorenzolapescara** [comment]
ci:forward

**Pull request closed without merging**